yarn-project/end-to-end/src/e2e_fees.test.ts

```diff
@@ -692,6 +696,74 @@ describe('e2e_fees', () => {
         initialSequencerGas + actualFee,
       ]);
     });
+
+    it('pays fee privately through FPC', async () => {
+      const mintedPrivateBananas = 10n;
+      // first, register the account with the PXE so that it can receive notes
+      await accountManager.register();
+
+      // right now, the only way to fund an undeployed account with private notes is to transfer them
```

Comment:
Seems like `privately_mint_private_note` should have a `recipient` argument, right?